src/index.ts

```diff
@@ -97,7 +166,7 @@ function addPoint(x: number, y: number) {
         points.splice(points.length - 1, 0, addedPoint);
 
         const startAndEndPoint = points[points.length - 1];
-        startAndEndPoint.segmentLength = magnitude(startAndEndPoint.x - x, startAndEndPoint.y - y);
+        startAndEndPoint.segmentLength = magnitude(startAndEndPoint.x - x, startAndEndPoint.y - y);

```

Comment:
I think the way to fix these issues with line endings is to have a .gitattributes in the repo to enable autocrlf.

Reply:
Resolved in https://github.com/jnm2/epic/commit/921618ed677e0f15069769ca39eb14fb6f90e495 with new .gitattributes file and `git add . --renormalize`.